### PR TITLE
Fixed Unit Tests/ Travis

### PR DESCRIPTION
**Pull request by MillerApps**
I fixed broken Unit test that were caused by the `NavigationController` class and as well the commenting out the old web3 code.

### CODE REVIEW

File: TrustTests/Factories/WalletSession.swift [hunk 1]
```diff
@@ -8,30 +8,30 @@ import BigInt
...
+            //web3: web3,
             balanceCoordinator: balance,
             nonceProvider: GetNonceProvider.make()
         )
     }
     static func makeWithEthBalance(
         account: Trust.Wallet = .make(),
         config: Config = .make(),
-        web3: Web3Swift = Web3Swift(),
+        //web3: Web3Swift = Web3Swift(),
```
Comment (vikmeup) [inline TrustTests/Factories/WalletSession.swift]: can you remove web3 ?

File: Gemfile.lock [hunk 2]
```diff
@@ -1,45 +1,46 @@
 GEM
   remote: https://rubygems.org/
   specs:
```
Comment (vikmeup) [inline Gemfile.lock]: why gems changed?

Comment (MillerApps) [reply Gemfile.lock]: Fixed Travis builds.

Comment (MillerApps) [reply Gemfile.lock]: I ran bundle update.

File: TrustTests/Factories/WalletSession.swift (hunk 1 above)
Comment (MillerApps) [reply TrustTests/Factories/WalletSession.swift]: Done

**Pull request merged**